app/src/main/java/com/example/makeiteven2/extras/AudioManager.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.makeiteven2.extras
+
+import android.content.Context
+import android.media.MediaPlayer
+import com.example.makeiteven2.R
+
+class AudioManager (context: Context) {
```

Comment:
why not like an object?

Reply:
because you only make it one time since we have only one activity